storage/replica_command.go

```diff
@@ -1426,13 +1426,18 @@ func (r *Replica) splitTrigger(batch engine.Engine, ms *engine.MVCCStats, split
 		return util.Errorf("unable to write MVCC stats: %s", err)
 	}
 
-	// Copy the last verification timestamp.
-	verifyTS, err := r.GetLastVerificationTimestamp()
+	// Copy the last replica GC and verification timestamps.
+	replicaGCTS, err := r.getLastReplicaGCTimestamp()
+	if err != nil {
+		return util.Errorf("unable to fetch last replica GC timestamp: %s", err)
+	}
+	if err := engine.MVCCPutProto(batch, nil /* &deltaMs */, keys.RangeLastReplicaGCTimestampKey(split.NewDesc.RangeID), roachpb.ZeroTimestamp, nil, &replicaGCTS); err != nil {
```

Comment:
@tschottdorf is right, if we're dealing with unreplicated keys, we drop the stats changes on the floor. This makes sense because there's really no way to store information about unreplicated data inside of replicated data. So the comments aren't needed, or if anything, could say something like `/* not tracked */`.